src/cephadm/cephadm

```diff
@@ -3548,6 +3549,33 @@ def install_sysctl(ctx: CephadmContext, fsid: str, daemon_type: str) -> None:
         call_throws(ctx, ['sysctl', '--system'])
 
 
+def sysctl_get(ctx: CephadmContext, variable: str) -> Union[str, None]:
+    """
+    Read a sysctl setting by executing 'sysctl -b {variable}'
+    """
+    out, err, code = call(ctx, ['sysctl', '-b', variable])
+    return out or None
+
+
+def filter_sysctl_settings(ctx: CephadmContext, lines: List[str]) -> List[str]:
+    """
+    Given a list of sysctl settings, examine the system's current configuration
+    and return those which are not currently set as described.
+    """
+    def testSetting(desiredLine: str) -> bool:
```

Comment:
`lower_case_with_underscores` is preffered both for function names and variables as recommended by PEP8 and to keep consistency with the rest of the file style in this case.

Reply:
you're absolutely right - been writing too much JS lately apparently